app/src/main/res/values/strings.xml

```diff
@@ -1366,4 +1368,20 @@
     <string name="voice_search_content_description">Voice search</string>
     <!-- Voice search prompt description displayed after the user presses the voice search button -->
     <string name="voice_search_explainer">Speak now</string>
+
+    <!-- Synced Tabs -->
+    <string name="synced_tabs_connect_to_sync_account">Connect with a Firefox Account.</string>
```

Comment:
All strings need a description above them to help our translators get a little more context about what they are about.